package.json

```
@@ -8,7 +8,10 @@
   "license": "AGPL-3.0-or-later",
   "scripts": {
     "tsc": "tsc",
-    "lint": "eslint *.js tests/*.test.js",
+    "format": "prettier . --check",
+    "format:fix": "prettier . --write",
+    "lint": "eslint .",
+    "lint:fix": "yarn format:fix && eslint . --fix",
```

How about this, focusing on just src and test
```suggestion
    "format": "prettier src/ tests/ --check",
    "format:fix": "prettier src/ tests/ --write",
    "lint": "eslint src/ tests/",
    "lint:fix": "yarn format:fix && eslint src/ tests/ --fix",
```

---

Is there a particular reason we'd want to avoid formatting other things? My thinking is that this is our source code, and non-formatted files should _generally_ be avoided, and so I'd rather make exclusions. With this approach we'd have to include every new file not in the `src` and `tests` directory too — `prettier src/ tests/ .github/ config/config.yaml html tsconfig.json test-setup.js README.md package.json docker-compose.yml CHANGES.md .pre-commit-config.yaml .eslintrc.cjs --check` seems unnecessary to me

---

I don't think we need to run prettier on any of those other files, they're rarely edited, and things like README and docker-compose don't need a specific format. Maybe we just start with this and reevaluate later?

---

I mean, I guess, but by that token why use an automated formatter at all? The whole point here is minimizing the amount of thinking we have to do in the future about formatting, so in my mind the more we can have auto-formatted the better, unless there's a specific reason not to format something — a file not being edited often doesn't mean you should have to think more about formatting than you do for other files

---

As discussed, updated ignores files and with `crawls`, `test-crawls`, `scratch` dir for misc scripts that are not usually checked in and should not be linted/prettified.